src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/CertificatePal.cs

```diff
@@ -360,6 +360,14 @@ public X500DistinguishedName IssuerName
             }
         }
 
+        public bool GetPolicyData(out byte[] ApplicationCertPoliciesData, out byte[] CertPoliciesData, out byte[] CertPo
+        licyMappingsData,
+                                   out byte[] CertPolicyConstraintsData, out byte[] EnhancedKeyUsageData, out byte[] InhibitAnyP
+                                   olicyExtensionData)
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
You have return true / return true / throw, and the only caller ignores the return value.  Why is the method boolean-returning?